plugin-hrm-form/src/utils/setUpMonitoring.js

```diff
@@ -84,11 +84,13 @@ function setUpRollbarLogger(plugin, workerClient, monitoringEnv) {
 function setUpFullStory() {
   FullStory.init({
     orgId: fullStoryId,
+    devMode: true,
```

Comment:
Will setting this to `true` all the time not disable Fullstory in deployed versions as well? Do we not need something like:

```suggestion
    devMode: process.env.NODE_ENV === 'development',
```

Reply:
According to the FullStory SDK docs, it sounds like setting devMode configuration to true is sufficient in order to turn off recording only in development environment. [https://github.com/fullstorydev/fullstory-browser-sdk](https://github.com/fullstorydev/fullstory-browser-sdk).

However, I also see that in one of the examples - Angular with devMode enabled - its contingent on env variable.  

If you think it is a better idea to implement devMode with `process.env.NODE_ENV === 'development'` as a condition, I will change it.

Reply:
Makes sense. Thank you, Steve.